src/transformers/reverse_path.test.ts

```diff
@@ -1,91 +1,95 @@
 import { describe, test, expect } from '@jest/globals';
 import { SVGPathData } from '../index.js';
 
+export function testReversePath(input: string) {
+  return new SVGPathData(input).reverse().encode();
+}
+
 describe('Reverse paths', () => {
   describe('Valid', () => {
     test('empty path', () => {
       const input = '';
       const expected = '';
-      expect(new SVGPathData(input).reverse().encode()).toEqual(expected);
+      expect(testReversePath(input)).toEqual(expected);
```

Comment:
in the other testX functions you introduced, you have the expect/toequal inside the helper method.

Also the other test files are in /tests, shouldn't this bet there as well?

Reply:
> expect/toequal inside the helper method

that was mistake than, vscode/intelij has hard time tracking errors if `expect` is in helper functions